ui/src/components/Header/Header.js

```diff
@@ -0,0 +1,28 @@
+import { user } from './Mock';
+
+export default function Header() {
+  return (
+    <header
+      className='py-2 px-8 h-20 flex items-center bg-rds-primary text-slate-100'
+    >
+      <img
+        src={user.picture.url}
+        className='rounded-full h-12 block mx-2'
+        alt='User Picture'
```

Comment:
alt tag should be empty here as the username is the description of the image which is next to the image itself.